app/helpers/application_helper/toolbar/physical_servers_center.rb

```diff
@@ -244,6 +244,8 @@ class ApplicationHelper::Toolbar::PhysicalServersCenter < ApplicationHelper::Too
             :url       => "provision",
             :send_checked => true,
             :url_parms => "main_div",
+            :enabled      => false,
+            :onwhen       => "1+",
```

Comment:
@matheuscmelo shouldn't this have been `:onwhen => "1"`
cc @mzazrivec

Reply:
The initial description says "1 or more physical server is selected". But I'd rather leave this to @matheuscmelo to explain.